lib/usd/utils/util.cpp

```diff
@@ -119,4 +120,21 @@ isInternalReference(const SdfPrimSpecHandle& primSpec)
     return isInternalRef;
 }
 
+bool 
+hasSpecs(const UsdPrim& prim)
+{
+    bool found{true};
+
+    UsdPrimCompositionQuery query(prim);
+
+    for (const auto& compQuaryArc : query.GetCompositionArcs()) {
+        if (!compQuaryArc.GetTargetNode().HasSpecs()) {
+            found = false; 
+            break;
+        }
+    }
+
+    return found;
```

Comment:
MAYA-103973: Renaming certain prims crashes Maya

@ppt-adsk The logic here checks to see if a node has really any specs to contribute to the final composite prim.  I am not sure 100% if this is the correct way of handling this case but that's the only way I found for now.

Please see my post on usd-interest around this:
https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/usd-interest/vweEUkTr6NY/MTYURcHfAgAJ

Reply:
@ppt-adsk all existing rename tests cases should pass with this change.

Reply:
I can't say I have any further experience...  Logic sounds reasonable to me.